crates/transparency/src/map/mod.rs

```diff
@@ -108,14 +108,12 @@ mod test {
 
     #[test]
     fn prove() {
-        fn check<D: SupportedDigest, K: VisitBytes + Clone, V: VisitBytes>(
+        fn check<D: SupportedDigest + std::fmt::Debug, K: VisitBytes + Clone, V: VisitBytes>(
```

Comment:
Confused why this is necessary when `Hash<D> where D: SupportedDigest` has a [`fmt::Debug` implementation](https://github.com/bytecodealliance/registry/blob/main/crates/crypto/src/hash/static.rs#L98).